src/views/Homepage.js

```diff
@@ -16,41 +29,102 @@ const StyledHero = styled.div`
   position: relative;
   width: 100%;
   height: 100vh;
-  min-height: 67.5em;
-  background-image: url(${props => props.src || "none"});
-  background-position: center;
+  background-image: url(${props => `${props.src}_mobile.jpg`}),
+    linear-gradient(to bottom, rgba(27, 38, 42, 1), rgba(84, 84, 92, 1));
+  background-position: center bottom;
   background-repeat: no-repeat;
   background-size: cover;
-  background-color: rgba(21, 37, 64, 1);
+  min-height: 45em;
+
+  @media only screen and (min-width: ${BETWEEN_SMALL_DEVICES_TABLET_UP}px) {
+    background-image: url(${props => `${props.src}.jpg`}),
+      linear-gradient(to bottom, rgba(27, 38, 42, 1), rgba(84, 84, 92, 1));
+    min-height: 55em;
+  }
+
+  @media only screen and (min-width: ${TABLET_PORTRAIT_UP}px) {
+    min-height: 65em;
+  }
+
+  @media only screen and (min-width: ${TABLET_LANDSCAPE_UP}px) {
+    min-height: 75em;
+  }
 `;
 
 const StyledMission = styled.div`
   position: absolute;
-  top: 42%;
-  left: 2.8em;
-`;
+  left: 50%;
+  transform: translate3d(-50%, 0, 0);
+  bottom: 65%;
 
-const StyledMissionTitle = styled.p`
-  font-size: 2.25rem;
-  font-weight: bold;
-  color: rgba(255, 255, 255, 1);
+  @media only screen and (min-width: ${TABLET_LANDSCAPE_UP}px) {
+    bottom: 75%;
+    transform: translate3d(-75%, 0, 0);
+  }
+
+  @media only screen and (min-width: ${DESKTOP_UP}px) {
+    transform: translate3d(-78%, 0, 0);
+  }
 `;
 
-const StyledMissionBody = styled.p`
-  font-size: 5rem;
+const StyledMissionTitle = styled.p`
   font-family: "Futura PT Extra";
```

Comment:
Maybe add a backup font/s in case the custom font fails to download? Like Helvetica and Arial?